src/simple-cache-client.ts

```diff
@@ -403,6 +417,38 @@ export class SimpleCacheClient {
     return await client.listRemoveValue(cacheName, listName, value);
   }
 
+  /**
+   * Retains slice of elements of a given list.
```

Comment:
I might expand on this a bit further to try to make it really clear that the rest of the list gets removed on the server-side.  Just to make it clear that we're not just retrieving the slice and leaving the server-side list unmodified.